specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1159,8 +1159,8 @@ namespace as their associated component. Custom serializers and
 deserializers, if supported, are treated in the same way as message
 handlers.
 
-The JAX-WS specification is available at
-_http://jcp.org/en/jsr/summary?id=224_ .
+The Jakarta XML Web Services specification is available at
+_https://jakarta.ee/specifications/xml-ws/2.3.2_ .
```

Comment:
I wouldn't use the version number in the directory name.  The previous reference was just to the top level JSR page.  If you quit after the xml-ws, then you would have a similar reference.  All of the available versions would then be listed.  It looks like this comment applies through out this section.  I'll only make this comment on this first instance and let you extrapolate.

Reply:
I haven't had time to do a full review of this but a few things caught my eye...

The JAX-WS spec (and JAXB, and SAAJ, and JWS) are *not* part of Jakarta EE,
and references to them should not be updated.